### PR TITLE
Added Polish language support.

### PR DESCRIPTION
**Pull request by L0um15**
During translation to Polish Language i noticed that Capital Ł letter is not displayed correctly. However i left "Enchantment Cracker" as default title. Here is example.
Adnotacja 2020-07-19 140950

### CODE REVIEW

**Comment by Earthcomputer** [comment]
The Ł issue is probably #145, it's weird and doesn't happen to everyone despite my attempts to fix it.

File: resources/i18n/EnchantmentCracker_pl.properties [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  \n\
+  Strona Github(Angielski):\n\
+  LINK https://github.com/Earthcomputer/EnchantmentCracker Earthcomputer/EnchantmentCracker\n\
+  \n\
+  Prosimy o zgłaszanie błędów w zakładce issue tracker w języku angielskim i załączeniu pliku o nazwie \n\
+  enchcracker.log
+
+# Enchantments
+# These should be the same as the official translations of enchantments, abbreviated where too long
+ench.protection = Ochrona.
```
Comment (Earthcomputer) [inline resources/i18n/EnchantmentCracker_pl.properties:70]: Are these enchantments supposed to have periods (.) at the end?

Comment (L0um15) [reply resources/i18n/EnchantmentCracker_pl.properties:70]: No. They are not necessary.

Comment (L0um15) [reply resources/i18n/EnchantmentCracker_pl.properties:70]: Only "ench.silk_touch = Jedwabny dot." has shortened word at the end

**Pull request merged**